src/components/flow/routers/localization/RouterLocalizationForm.tsx

```diff
@@ -194,7 +198,11 @@ export default class RouterLocalizationForm extends React.Component<
 
     const hasCasesWithArguments = !!this.state.cases.find((kase: Case) => {
       const orginalCase = getOriginalCase(this.props.nodeSettings, kase.uuid) as Case;
-      return orginalCase.arguments && orginalCase.arguments.length > 0;
+      return (
+        orginalCase.arguments &&
+        orginalCase.arguments.length > 0 &&
+        orginalCase.type !== Operators.has_number_between
```

Comment:
This one-liner would be more robust:
`(originalCase.arguments || []).length == 1`